src/ray/gcs/gcs_client/service_based_accessor.h

```diff
@@ -95,15 +97,12 @@ class ServiceBasedActorInfoAccessor : public ActorInfoAccessor {
       const ActorID &actor_id,
       const OptionalItemCallback<rpc::ActorCheckpointIdData> &callback) override;
 
- protected:
-  ClientID subscribe_id_;
-
  private:
   ServiceBasedGcsClient *client_impl_;
 
-  typedef SubscriptionExecutor<ActorID, ActorTableData, ActorTable>
-      ActorSubscriptionExecutor;
-  ActorSubscriptionExecutor actor_sub_executor_;
+  /// Subscribe to GCS table information.
+  GcsPubSub gcs_sub_;
```

Comment:
I think we can just use one `GcsPubSub` instance (which should be owned by the gcs client) for different accessors.

Reply:
fixed